source/_components/remote.xiaomi_miio.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: page
+title: "Xiaomi IR Remote"
+description: "Instructions for how to integrate the Xiaomi IR Remote within Home Assistant."
+date: 2017-01-25 17:08
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: xiaomi.png
+ha_category: Remote
+ha_release: 0.62
+ha_iot_class: "Local Polling"
+---
+
+The `xiaomi miio` remote platform allows you to send IR commands from your Xiaomi IR Remote (ChuangmiIr).
+
+Please follow the instructions on [Retrieving the Access Token](/components/vacuum.xiaomi_miio/#retrieving-the-access-token) to get the API token to use in the `configuration.yaml` file.
+
+## {% linkable_title Configuring the Platform %}
+
+To add a Xiaomi IR Remote to your installation, add the following to your configuration.yaml file:
+```yaml
+remote:
+  - platform: xiaomi_miio
+    name: "bathroom remote"
+    host: 192.168.42.42
+    token: YOUR_TOKEN
+    slot: 1
+    timeout: 30
+    hidden: false
+    commands:
+      activate_towel_heater:
+        command:
+          - base64 encoded string learned from remote
+      read_bad_poem:
+        command:
+          - base64 encoded string learned from remote
+          - base64 encoded string learned from remote
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration